packages/compass-sidebar/src/components/navigation-items.tsx

```diff
@@ -225,34 +245,57 @@ export function NavigationItems({
 
   return (
     <>
-      <NavigationItem<''>
-        isExpanded={isExpanded}
-        onAction={onAction}
-        glyph="CurlyBraces"
-        label="My Queries"
-        tabName="My Queries"
-        isActive={currentLocation === 'My Queries'}
-      />
-      {showPerformanceItem && (
-        <NavigationItem<''>
-          isExpanded={isExpanded}
-          onAction={onAction}
-          glyph="Gauge"
-          label="Performance"
-          tabName="Performance"
-          isActive={currentLocation === 'Performance'}
-        />
-      )}
-      <NavigationItem<DatabasesActions>
-        isExpanded={isExpanded}
-        onAction={onAction}
-        glyph="Database"
-        label="Databases"
-        tabName="Databases"
-        actions={databasesActions}
-        isActive={currentLocation === 'Databases'}
-        showTooManyCollectionsInsight={showTooManyCollectionsInsight}
-      />
+      <ContentWithFallback
+        isContentReady={!!isReady}
```

Comment:
Noticed we're getting some flake in CI because Performance navigation item is hidden by default and test runner will often click Performance tab when trying to click Databases, so decided that the most straightforward thing to do here would be to hide these until instance info is fetched